### PR TITLE
[Issue #95707] Remove WindowsGMSA and WindowsRunAsUserName features from code

### PR DESCRIPTION
**Pull request by mrmassis**
Remove the WindowsGMSA, WindowsRunAsUserName and traces from code. Fix some entries that were altereds.

/kind cleanup

### CODE REVIEW

File: pkg/kubelet/kuberuntime/kuberuntime_container_windows.go [hunk 1]
```diff
@@ -128,11 +128,6 @@ func (m *kubeGenericRuntimeManager) generateWindowsContainerConfig(container *v1
 	if username != "" {
 		wc.SecurityContext.RunAsUsername = username
 	}
-	if utilfeature.DefaultFeatureGate.Enabled(kubefeatures.WindowsGMSA) &&
```
Comment (ialidzhikov) [inline pkg/kubelet/kuberuntime/kuberuntime_container_windows.go:131]: Why you remove the whole if statement? Shouldn't you remove only the `tilfeature.DefaultFeatureGate.Enabled(kubefeatures.WindowsGMSA)` part? @marosset , correct me if I am wrong

Comment (mrmassis) [reply pkg/kubelet/kuberuntime/kuberuntime_container_windows.go:131]: Yes, you are correct. How i do to revert it in this PR? Should I open other PR?

File: pkg/features/kube_features.go [hunk 2]
```diff
@@ -206,7 +206,6 @@ const (
 	// alpha: v1.10
 	//
 	// Enable Hyper-V containers on Windows
-	// Deprecated in 1.20 and removed in 1.21
```
Comment (SergeyKanzhelev) [inline pkg/features/kube_features.go:209]: why this change?

File: pkg/features/kube_features.go [hunk 3]
```diff
@@ -297,7 +296,7 @@ const (
 	CSIStorageCapacity featuregate.Feature = "CSIStorageCapacity"
 
 	// owner: @alculquicondor
-	// beta: v1.20
```
Comment (SergeyKanzhelev) [inline pkg/features/kube_features.go:300]: why this change?

File: pkg/kubelet/kuberuntime/kuberuntime_container_windows.go (hunk 1 above)
Comment (SergeyKanzhelev) [reply pkg/kubelet/kuberuntime/kuberuntime_container_windows.go:131]: You can continue in this PR. Just keep editing files in this branch. And then squash your commits. See https://www.kubernetes.dev/docs/guide/github-workflow/#squash-commits

File: staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/internal/testdata/swagger.json [hunk 4]
```diff
@@ -3678,20 +3678,7 @@
     },
     "io.k8s.api.core.v1.WindowsSecurityContextOptions": {
       "description": "WindowsSecurityContextOptions contain Windows-specific options and credentials.",
-      "properties": {
```
Comment (SergeyKanzhelev) [inline staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/internal/testdata/swagger.json:3681]: have you regenerated this file or edited manually? I think you need to fix types.go file and regenerate swagger `make update`

Comment (mrmassis) [reply staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/internal/testdata/swagger.json:3681]: Hi,
How can i do it? Could you give an exemple?
